mta-sts.md

```diff
@@ -498,4 +498,34 @@ https://mta-sts.example.com/.well-known/mta-sts.json:
 
 ~~~~~~~~~
 
+# Appendix 2: Sending MTA State Machine
+
+It is helpful to think of the different states of a sending MTA and a receiving
```

Comment:
I dunno. I wasn't thrilled with it, either, but I had the same feeling:
better to over-explain than under. But is it even clear? Dunno.

On Mon, Sep 12, 2016 at 2:11 PM abrotman <notifications@github.com> wrote:

> In mta-sts.md
> <https://github.com/mrisher/smtp-sts/pull/103#discussion_r78360211>:
>
> > @@ -498,4 +498,34 @@ https://mta-sts.example.com/.well-known/mta-sts.json:
> >
> >  ~~~~~~~~~
> >
> > +# Appendix 2: Sending MTA State Machine
> > +
> > +It is helpful to think of the different states of a sending MTA and a receiving
>
> I also find it wordy, but given some of the feedback we've received from
> others about expanding various definitions, I don't think it is
> unnecessarily so. I'd say that as long it makes sense (which it seems to
> do), we let it go like this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mrisher/smtp-sts/pull/103/files/ea1fb592e4efc3ad5d1a840d6e59ea510d953f46#r78360211>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1vixcYz_6vmzGAhwPfOdh-NaZfUq63ks5qpUFcgaJpZM4J6C5r>
> .
>